CuratedContent/psip-podcast.md

```diff
@@ -0,0 +1,32 @@
+# What makes PSIP suitable for Exascale Computing Project?
+
+The Exascale Computing Project focusses on the development of a highly capable exascale ecosystem. With hundreds of people resources and codes of different magnitudes involved in this huge project, what is the path for software sustainabiltiy and lasting software quality?.
+
+
+Resource information | Details 
+:--- | :--- 
+Podcast title  | Productivity Sustainability Improvement Planning (PSIP) for ECP software
+Presenters | [Elaine Raybourn](https://github.com/elaineraybourn)
+Web links | [Podcast link at Exascaleproject.org ](https://exascaleproject.org/method-enables-collaborative-software-teams-to-enhance-effectiveness-and-efficiency/)
+
+
+
+This [interview podcast](https://exascaleproject.org/method-enables-collaborative-software-teams-to-enhance-effectiveness-and-efficiency/ "PSIP for ECP") of [Elaine Raybourn](https://github.com/elaineraybourn "Elaine Raybourn Profile") from the [ECP IDEAS](https://ideas-productivity.org) team gives an excellent insight on how the Productivity Sustainability Improvement Planning (PSIP) software improvement process is beneficial for Exascale Computing Project (ECP) software and application teams.
```

Comment:
I started review but decided to just submit PR with my changes. Nonetheless, here are my review comments.

- Best if keep line length to < 80
- `ECP IDEAS` is backwards. Should be `IDEAS-ECP`
- Missing `and` between `Productivity` and `Sustainability` in a couple of places.
- Missing `the` before `Exascale Computing Project`
  - Why spell out ECP here when alread done in first paragraph?

Reply:
To continue discussion, please refer to new PR #463